client/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
Same here, please customise these fields

Reply:
"short_name": "INSEE Brut"
"name": "Mirror site for INSEE data aimed to be exhaustive, faster and easier to browse."